### ISSUE TITLE
project status

### ISSUE THREAD

**Issue by gabrielflorit** [newsdev/elex#145]
Hey all,

What's the status of *elex* - is it production ready? What is left to do, and how can we help?

Thanks for open-sourcing this - end the elections arms race indeed.

**Comment by eads** [comment]
It depends on what you mean by "production ready" :)

Is it being used in production? yes! The NYT has already been using it for
real election results.

Is it ready to be used in production everywhere? I'd say we're close. But
lots of little things are outstanding and I'm confident on principle we've
still got a few serious bugs to uncover.

For a sense of what needs to be done, check out the issue queue. The next
milestone is 0.2.0 and we'd like to hit 1.0 by mid-January for the Iowa
caucus.

What's more pressing is an open source system to get results to the web --
basically a UI to handle calling races, setting ballot order, overriding
names, etc. Me/NPR is gonna be working on that (probably in django but I'm
open to trying something like node) in the coming weeks and I'd be into
some collaboration there if you're interested.

**Comment by gabrielflorit** [comment]
Thanks @eads, good to know.
